js/bitbay.js

```diff
@@ -286,9 +498,30 @@ module.exports = class bitbay extends Exchange {
     sign (path, api = 'public', method = 'GET', params = {}, headers = undefined, body = undefined) {
         let url = this.urls['api'][api];
         if (api === 'public') {
-            let query = this.omit (params, this.extractParams (path));
-            url += '/' + this.implodeParams (path, params) + '.json';
-            url += '?' + this.urlencode (query);
+            const query = this.omit (params, this.extractParams (path));
+            url += '/' + this.implodeParams (path, params) + '.json' + '?' + this.urlencode (query);
+        } else if (api === 'v1_01Public') {
+            const query = this.omit (params, this.extractParams (path));
+            url += '/' + this.implodeParams (path, params);
+            if (Object.keys (query).length) {
+                url += '?' + this.urlencode (query);
+            }
+        } else if (api === 'v1_01Private') {
+            this.checkRequiredCredentials ();
+            const query = this.omit (params, this.extractParams (path));
+            url += '/' + this.implodeParams (path, params);
+            if (Object.keys (query).length) {
+                url += '?' + this.urlencode (query);
+            }
+            const nonce = this.now ();
+            const payload = this.apiKey + nonce + (body ? this.json (body) : '');
```

Comment:
This line was breaking it, apparently... the transpiler cannot handle too complex one-liners, so, it has to broken apart:

```JavaScript
let payload = this.apiKey + none;
if (body !== undefined) {
    payload += this.json (body);
}
```

Apart from that there was a `null` instead of `undefined` here: https://github.com/ccxt/ccxt/pull/5208/files#diff-6af658a1c4be62c2589caa23613020b5R393

Will upload a fixed version shortly.